content/en/real_user_monitoring/reactnative/troubleshooting.md

```diff
@@ -0,0 +1,173 @@
+---
+title: React Native Web View Tracking
```

Comment:
I think the title has not be changed, should it be "Troubleshooting"?

Reply:
Oof, you're right, thanks!